theme/js/components/utils/read-more.vue

```diff
@@ -72,8 +72,8 @@ export default {
       let contentHeight = Array.from(this.$refs.container.children)
       .map(getHeight)
       .reduce((total, height) => total + height, 0)
-      this.readMoreRequired = contentHeight > getHeight(this.$refs.container);
       this.containerHeight = DEFAULT_HEIGHT;
+      this.readMoreRequired = Math.round(contentHeight) > this.containerHeight;
```

Comment:
Seems like it would work as well?
```suggestion
      this.readMoreRequired = contentHeight  > this.containerHeight;
```

Reply:
yes ! This was from my previous attempt